plugins/pulp_rpm/plugins/distributors/yum/metadata/updateinfo.py

```diff
@@ -122,6 +122,12 @@ def add_unit_metadata(self, item, filtered_pkglist):
         if erratum_unit.reboot_suggested:
             self.xml_generator.completeElement('reboot_suggested', {}, 'True')
 
+        if erratum_unit.relogin_suggested:
```

Comment:
This adds `relogin_suggested` to the top level of erratum xml. As per [SUSE docs](https://en.opensuse.org/openSUSE:Standards_Rpm_Metadata_UpdateInfo), it's only inside of `package`.

`reboot_suggested` is added both to the top level and inside the `package` tag because of how certain versions of `yum` handles that, iirc. So I suggest to comply with SUSE docs for its tags.